style_editor.html

```diff
@@ -9475,19 +9479,28 @@
 
   // Set up the event listener for the menu
   menu.addEventListener('change', function() {
-    // Get the selected option value
-    const value = menu.value;
-
     // If the selected value is not the default, enable the button and set its action
-    if (value !== '') {
+    if (menu.value !== '') {
       button.disabled = false;
-      button.onclick = actions[value];
+      button.onclick = actions[menu.value];
```

Comment:
Seems like it would be simpler to just have the onclick be AddClickedEffect() all the time.

Reply:
ah. sure, now that that exists. how about now?

Reply:
"simpler to just have the onclick be AddClickedEffect() all the time"
proving to not be simpler. 
could you clarify?